Week1/MAKEME.md

```diff
@@ -35,33 +35,106 @@ Write commands to do following:
+  // TODO -> here you declare your variable
+  console.log('the value of x will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of x
+  // TODO -> here you initialize your variable
+  console.log('the value of x will be: whateverYouThinkItWillLog')
+  // TODO -> log value of x again
+  ```
+
+2. Declare a variable `y` and assign a string to it.
+  2.1 Write a console.log statement in which explain what you think the value of the string is
+  2.2 Now console.log the variable `y`.
+  2.3 Now assign a new string to the variable `y`
+  2.4 Just like you did before write a console.log statement that explains what you think will be logged to the console.
+  2.5 Now console.log `y` again.
+  ```js
+  // TODO -> here you declare AND assign your string
+  console.log('the value of my string will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of the string to the console
+  // TODO -> assign a new value to your variable x
+  console.log('the value of my string will be: whateverYouThinkItWillLog')
+  // TODO -> log the actual value of the string to the console
+  ```
+
+3. How do you round the number 7.25, to the nearest integer?
+  3.1 Declare a variable `z` and assign the number 7.25 to it.
+  3.2 Console.log `z`.
+  3.3 Now round `z` to the nearest integer.
+  3.4 Console.log `z` again.
+
+4. Arrays!
+  4.1 Declare an empty array (you can decide on how to call it yourself).
+  4.2 Write a console.log statement in which explain what you think the value of the array is.
+  4.3 Console.log your array.
+  4.4 Create an array that has your favorite animals inside
+  4.5 Log your array
+  4.6 Ads Daan's favorite animal (baby pig) to the existing array
+  4.7 Log your new array!
+  4.8 Now add Gijs's favorite animal to the array (moose), but make sure it will be placed after the first animal that you have placed in your array.
+  4.9 Write a console.log statement in which explain what _you think_ the new value of the array is.
+  4.10 Log your new new array!
+  4.11 Log the length of the array, add a message: "The array has a length of: "(here you should show the length of the array)
+  4.12 Eyad does not like giraffes, delete this animal from the array
+  4.13 Again log your new array.
+  4.14 Now if unlike Gijs, you don't like mooses and you want to delete it from the array, but you don't know the position or the `index` of the item in the array, how can you find it?
+  4.15 Log the index of moose to the console. Add a message so it says: "The item you are looking for is at index: " (here you should show the index of the item)
+
+5. More strings 
+  5.1 Let's consider the following string: `let mystring = "this,is,a,test"`.
+  5.2 Add the string to your file and console.log it.
+  5.3 Find a way to get the length of `mystring`.
+  5.4 Console.log the length of `mystring`.
+  5.5 The comma's make that the sentence is quite hard to read. Find a way to remove the comma's from the sting and replace them with a spaces
```

Comment:
typo: sting -> string. I would replace the commas with `myString = myString.replace(/,/g, ' ')` But that would mean we need explain what a regular expression and what the global flag does. BTW, I propose we hammer in the use of camelCase early, so can we make it `myString`?

Reply:
yes I agree, again also got a big carried away here, i moved half of the assignment to week3